bazel_drake_cmake_external/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+# -*- cmake -*-
+
+# Copyright (c) 2017 Daniel Stonier
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+cmake_minimum_required(VERSION 3.5.1)
+project(bazel_drake_cmake_external)
+
+find_program(BAZEL_EXECUTABLE bazel)
+
+if(NOT BAZEL_EXECUTABLE)
+  message(FATAL_ERROR "Could NOT find bazel executable")
+endif()
+
+if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+  set(BAZEL_COMPILATION_MODE dbg)
+else()
+  set(BAZEL_COMPILATION_MODE opt)
+endif()
+
+set(BAZEL_ENV CC=${CMAKE_C_COMPILER} CXX=${CMAKE_CXX_COMPILER})
+set(BAZEL_ARGS --compilation_mode=${BAZEL_COMPILATION_MODE} --config=unsupported_crosstool --cxxopt=-std=c++1y)
+
+if(CMAKE_VERBOSE_MAKEFILE)
+  set(BAZEL_ARGS --subcommands ${BAZEL_ARGS})
+endif()
+
+include(ExternalProject)
+
+ExternalProject_Add(drake
+  URL https://github.com/RobotLocomotion/drake/archive/93094a7cf89212fcd7a2ac0701469bcf3f083178.tar.gz
+  URL_HASH SHA256=5b7c0e9af659551367206ed6ebb3dde677af73d4a3fca3dbb084d277b18a44b6
+  TLS_VERIFY 1
+  CONFIGURE_COMMAND :
+  BUILD_COMMAND ${BAZEL_ENV} ${BAZEL_EXECUTABLE} build ${BAZEL_ARGS} ...
+  BUILD_IN_SOURCE 1
+  BUILD_ALWAYS 1
+  INSTALL_COMMAND ${BAZEL_ENV} ${BAZEL_EXECUTABLE} run ${BAZEL_ARGS} install -- ${CMAKE_INSTALL_PREFIX}
+  USES_TERMINAL_BUILD 1
+  USES_TERMINAL_INSTALL 1
+)
+
+ExternalProject_Add(drake_shambhala
```

Comment:
Oh, explicitly wanting to demonstrate how to pull in drake, and a dependency that depends on drake?